examples/qa/assets/js/script.js

```diff
@@ -0,0 +1,196 @@
+var app = new Vue({
+  el: "#app",
+  data: {
+    envId: "",
+    apiKey: "",
+    timeout: 2000,
+    pollingInterval: 60000,
+    segmentApiKey: "",
+    bucketing: true,
+    visitorId: "test-visitor",
+    anonymousId: false,
+    isAuthenticated: false,
+    newVisitorId: "",
+    context: "{\n}",
+    envOk: false,
+    envError: null,
+    visitorOk: false,
+    visitorError: null,
+    visitorAuth: {},
+    visitorUnauth: {},
+    eventOk: false,
+    eventError: null,
+    data: null,
+    hit: { t: "EVENT" },
+    hitTypes: ["EVENT", "TRANSACTION", "ITEM", "PAGE", "SCREEN"],
+    flag: { name: "", type: "bool", defaultValue: "", activate: true },
+    flagOk: false,
+    flagInfo: { name: "" },
+    flagInfoOk: false,
+  },
+  methods: {
+    getEnv() {
+      this.$http.get("/currentEnv").then((response) => {
+        // get body data
+        this.currentEnv = response.body;
+        this.bucketing = response.body.bucketing;
+        this.envId = response.body.env_id;
+        this.apiKey = response.body.api_key;
+        this.timeout = response.body.timeout;
+        this.pollingInterval = response.body.pollingInterval;
+        this.segmentApiKey = response.body.segment_api_key;
+      });
+    },
+    setEnv() {
+      this.envOk = false;
+      this.envError = null;
+      this.$http
+        .post("/setEnv", {
+          environment_id: this.envId,
+          api_key: this.apiKey,
+          bucketing: this.bucketing,
+          timeout: this.timeout || 0,
+          polling_interval: this.pollingInterval || 0,
+          segment_api_key: this.segmentApiKey,
+        })
+        .then(
+          (response) => {
+            // get body data
+            this.someData = response.body;
+            this.envOk = true;
+          },
+          (response) => {
+            this.envOk = false;
+            this.envError = response.body;
+          }
+        );
+    },
+    setVisitor() {
+      this.visitorOk = false;
+      this.visitorError = null;
+      this.data = null;
+
+      this.$http
+        .post("/setVisitor", {
+          visitor_id: this.visitorId,
+          context: this.context ? JSON.parse(this.context) : null,
+        })
+        .then(
+          (response) => {
+            // get body data
+            this.data = response.body;
+            this.visitorId = response.body.visitorId;
+            this.anonymousId = response.body.anonymousId;
+            this.visitorOk = true;
+            console.log('youpi')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/flagship-io/flagship-go-sdk/pullRequest?prid=8335991)
```suggestion
            console.log("youpi")
```

Reply:
merged